### DISCUSSION TITLE
Saving to missionProfileNamespace

### DISCUSSION THREAD

**Discussion by VyMajoris** [Vdauphin/HeartsAndMinds#1440] [General]
[Has any though given to this?](https://community.bistudio.com/wiki/missionProfileNamespace)

Saving to the missionProfileNamespace would keep all persistent data off the server profile, which is better for performance.

**Comment by Vdauphin** [comment]
Hello,

> [Has any though given to this?](https://community.bistudio.com/wiki/missionProfileNamespace)

I consider it but didn't see any advantage. It could be only usefull if you frequently migrate the dedicated server without keeping cba setting which is not something common

> which is better for performance.

There is no performance improvement

Cheers
